support-frontend/assets/pages/digital-subscription-checkout/checkoutFormIsSubmittableActions.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { getErrors, getFormFields } from 'pages/digital-subscription-checkout/digitalSubscriptionCheckoutReducer';
+import type { State } from './digitalSubscriptionCheckoutReducer';
+
+// ----- Functions ----- //
+
+const enableOrDisablePayPalExpressCheckoutButton = (formIsSubmittable: boolean) => {
```

Comment:
maybe there's a way of factoring these out as it's shared with the contributions page. but maybe it's not worth the effort

Reply:
I tried that initially but it caused such a mess that I went down this path.